### PR TITLE
fix: markdown command undo bug

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's the right issue type on the title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has a description of the breaking change

### Description
* fixed markdown command and keymap bug when executing undo.

ex1) the next content is removed when undo the command
**as-is**

**to-be**

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/editor/src/__test__/unit/markdown/util.ts [hunk 1]
```diff
@@ -45,3 +46,11 @@ export function createHTMLrenderer() {
 
   return customHTMLRenderer;
 }
+
+export class TestEditorWithNoneDelayHistory extends MarkdownEditor {
+  get defaultPlugins() {
+    const plugins = [...this.keymaps, history({ newGroupDelay: -1 })];
+
+    return plugins;
```
Comment (jung-han) [inline apps/editor/src/__test__/unit/markdown/util.ts:54]: 바로 반환해도 될 것 같습니다 ㅎ

Comment (jungeun-cho) [inline apps/editor/src/__test__/unit/markdown/util.ts:54]: `[...this.keymaps, history({ newGroupDelay: -1 })];`
바로 리턴할수있어요~

**Review by jungeun-cho — APPROVED** [review]
수고하셨습니다.

File: apps/editor/src/markdown/marks/thematicBreak.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import { DOMOutputSpecArray } from 'prosemirror-model';
+import type { Transaction } from 'prosemirror-state';
```
Comment (jung-han) [inline apps/editor/src/markdown/marks/thematicBreak.ts:2]: 사용하는 건 처음봤네요 ㅋㅋ 신기하네요

**Review by jung-han — APPROVED** [review]
리뷰 완료합니다. 상당히 로직이 복잡하네요...고생하셨습니다!

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Pull request merged**